test/e2e/framework.go

```diff
@@ -63,7 +63,7 @@ import (
 var AntreaConfigMap *corev1.ConfigMap
 
 const (
-	defaultTimeout  = 90 * time.Second
+	defaultTimeout  = 180 * time.Second
```

Comment:
Not sure if will extend the execution time when there is something wrong, why it needs to be doubled?

Reply:
It may need to be extended since we found some case failure when defaultTimeout=90. We will update this time according to our test result.

Reply:
I suspect that it's because the NFS storage is much slower than vSAN. The K8s deployment time in vSAN is at least 2 times faster than NFS. I think maybe we don't have enough time root cause today.  As I discussed with Shuyang, we observed less random case failures with larger timeout. If we don't merge this change today, we cannot migrate the CI.  Is it OK to merge this change with 180s timeout and later we have some follow-up patches to fix the root cause?